vertx-codegen-protobuf/src/main/java/io/vertx/codegen/protobuf/annotations/ProtobufGen.java

```diff
@@ -18,9 +26,10 @@
  *   public static void toProto(...);
  * }
  * </pre>
- * Note: This annotation only works with the `@DataObject` annotation.
+ * Note: This annotation only works with the {@literal @}DataObject annotation.
```

Comment:
Actually `{@literal @DataObject}` I think

Reply:
Both methods work, but I guess `{@literal @DataObject}` is more idiomatic